src/main/java/cms/gov/madie/measure/resources/MeasureTransferController.java

```diff
@@ -196,4 +201,38 @@ private void setMeasureElmJsonAndErrors(Measure measure, String apiKey, String h
       measure.setCqlErrors(true);
     }
   }
+
+  protected List<Group> reorderGroupPopulations(List<Group> groups) {
```

Comment:
How is reordering groups helping us in importing Test Cases with DENOM Exclusion ?

Reply:
@RohitKandimalla Because there is an order of populations, the order is in TestCaseServiceUtil.EXPECTED_VALUE_ORDER